src/cobra/test/test_util/test_util.py

```diff
@@ -1,9 +1,37 @@
-"""Test functions of util.py"""
+"""Test functions of util.py ."""
 
-from cobra.util import show_versions
+from cobra.util import AutoVivification, format_long_string, show_versions
 
 
-def test_show_versions(capsys):
+def test_format_long_string_with_long_string() -> None:
+    """Test functionality by formatting a long string."""
+    long_string = (
+        "This is a really long string, but is it long enough."
+        "I hope it is long enough so that format_long_string() function works."
+    )
+    expected_output = "This is a really long string, but is it long en..."
+    assert expected_output == format_long_string(long_string)
+
+
+def test_format_long_string_with_short_string() -> None:
+    """Test functionality by formatting a short string."""
+    short_string = "This is short string."
+    assert short_string == format_long_string(short_string)
+
```

Comment:
You could work with `@pytest.mark.parametrize` to condense these two test definitions into one and simply test different instances.

Reply:
Yes I'll update it.